skaha/src/main/java/org/opencadc/skaha/session/SessionAction.java

```diff
@@ -315,10 +315,15 @@ public Session getSession(String forUserID, String sessionID) throws Exception {
         throw new ResourceNotFoundException("session " + sessionID + " not found");
     }
 
-    public List<Session> getAllSessions(String forUserID) throws Exception {
+    @Deprecated
```

Comment:
We are adding a deprecated `_getUserSession` method? Add a detailed JavaDoc for this?

Reply:
Removing...